### ISSUE TITLE
[v2] theme tag being set

### ISSUE THREAD

**Issue by Nick-bcm** [JedWatson/react-select#3112]
**Version** 2.1.0
**React** 15.6.2

When react select is being rendered it throws this warning:
```
Warning: Unknown prop `theme` on <input> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in input (created by AutosizeInput)
    in div (created by AutosizeInput)
    in AutosizeInput (created by Input)
    in div (created by Input)
    in Input (created by Select)
    in div (created by ValueContainer)
    in ValueContainer (created by Select)
    in div (created by Control)
    in Control (created by Select)
    in div (created by SelectContainer)
    in SelectContainer (created by Select)
```

See console on load/render:
[![Edit ykm2z7o441](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/ykm2z7o441)

When I tried to view the code of [Input.js](https://github.com/JedWatson/react-select/blob/master/src/components/Input.js#L18), I found out that the `inputCSS` function does not receive `isDisabled` in any case. I think it's component need to be refactored.

**Comment by amorales26** [comment]
I am also having this issue. This needs to be addressed !

**Comment by miuosh** [comment]
Version 2.1.0
React 15.6.2

I have this issue after update react-select to ver. 2

**Comment by casvil** [comment]
Same here

**Comment by j-funk** [comment]
Also having this issue. Cluttering up the console.

**Comment by vijayranghar** [comment]
any workaround for this @Nick-bcm

**Comment by joelstransky** [comment]
Just got hit with this. Seems like that prop should be renamed to `data-theme`

**Comment by Nick-bcm** [comment]
> any workaround for this @Nick-bcm

No. Just waiting for any solution.

**Comment by joelstransky** [comment]
I confirmed that changing the prop to `data-theme` fixes the problem. Now I just have to verify that I didn't just break theming and set up a PR

**Comment by joelstransky** [comment]
Turns out this bug was emanating from another package [react-input-autosize](https://github.com/JedWatson/react-input-autosize)
I submitted a [pull-request](https://github.com/JedWatson/react-input-autosize/pull/146) to blacklist `"theme"` as well.

**Comment by gwyneplaine** [comment]
@Nick-bcm  thanks for raising this, resolved in v2.1.1 
https://codesandbox.io/s/pyjy2w593x
